fido/fido.py

```diff
@@ -118,23 +128,21 @@ def detect_formats(self):
 
         return results
 
+
 class Fido:
-    def __init__(self, quiet=False, bufsize=None, container_bufsize = None, printnomatch=None, printmatch=None, zip=False, nocontainer=False, handle_matches=None, conf_dir=None, format_files=None, containersignature_file=None):
+    def __init__(self, quiet=False, bufsize=None, container_bufsize=None, printnomatch=None, printmatch=None, zip=False, nocontainer=False, handle_matches=None, conf_dir=None, format_files=None, containersignature_file=None):
         global defaults
         self.quiet = quiet
-        self.bufsize = (defaults['bufsize'] if bufsize == None else bufsize)
-        self.container_bufsize = (defaults['container_bufsize'] if container_bufsize == None else container_bufsize)
-        self.printmatch = (defaults['printmatch'] if printmatch == None else printmatch)
-        self.printnomatch = (defaults['printnomatch'] if printnomatch == None else printnomatch)
-        self.handle_matches = (self.print_matches if handle_matches == None else handle_matches)
+        self.bufsize = (defaults['bufsize'] if not bufsize else bufsize)
+        self.container_bufsize = (defaults['container_bufsize'] if not container_bufsize else container_bufsize)
+        self.printmatch = (defaults['printmatch'] if not printmatch else printmatch)
+        self.printnomatch = (defaults['printnomatch'] if not printnomatch else printnomatch)
+        self.handle_matches = (self.print_matches if not handle_matches else handle_matches)
```

Comment:
Can any of these be falsey values that are not None? Should those cases be treated differently?